packages/itinerary-body/src/stories/OtpRrItineraryBody.story.tsx

```diff
@@ -49,12 +49,14 @@ interface StoryWrapperProps {
   itinerary: Itinerary;
   showRouteFares: boolean;
   TimeColumnContent: FunctionComponent<TimeColumnContentProps>;
+  shouldAlwaysCollapseAlerts: boolean;
```

Comment:
Sort props (case insensitive, 2 instances)

Reply:
Thanks for that - updated.